package.json

```diff
@@ -4,8 +4,10 @@
   "description": "Wrapper to run nemo/mocha suites",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && ./bin/nemo-runner -B test -G @suite1",
-    "test:parallel": "./bin/nemo-runner -B test -G @suite1,@suite2,@suite3,@suite4,@suitex -P chrome,firefox",
+    "test": "npm run lint && npm run nemo",
+    "nemo": "DEBUG=nemo-runner* ./bin/nemo-runner -B test -G @suite1",
+    "nemo:d": "./bin/nemo-runner -B test -G @suite1 --debug-brk",
```

Comment:
I tried running `npm run nemo:d` and it failed due to the fact that I'm running node 8.  Not a big deal, just calling it out.

Reply:
Oh right. Yeah the flags change in node8.. need to account for that in the code itself as well. Filing an issue for that. Thanks!